src/conceptual/handling-sensitive-data.mdx

```diff
@@ -193,32 +215,110 @@ Note also, that there is a small workaround that makes  \`slSensitive\` property
+      android: {
+      java: `
+Smartlook.registerBlacklistedClass(WebView.class);
+`,
+      kotlin: `
+Smartlook.registerBlacklistedClass(WebView::class.java)
+`
+    },
+      ios: {
+      swift: `
+Smartlook.registerBlacklisted(object: WKWebView.class)
+      `,
+      'objective-c': `
+[Smartlook registerBlacklistedObject:WKWebView.class];
+      `,
+    },
+    react: {
+      js: `
+// This functionality is not available on this platform.
+`,
+    },
+xamarin: {
+      csharp: `
+Smartlook.Analytics.RegisterWhitelistedObject(WKWebView.Class);
+      `,
+    },
+    cordova: {
+      html: `
+// This functionality is not available on this platform.
+`,
+    },
+    ionic: {
+      html: `
+// This functionality is not available on this platform.
+      `,
+    },
+    unreal: {
+      csharp: `
+// This functionality is not available on this platform.
+      `,
+    },
+  unity: {
+    csharp: `
+// This functionality is not available on this platform.
+    `,
+  },
+    flutter: {
+      dart: `
+// This functionality is not available on this platform.
+      `,
+  },
+    cocos: {
+      java: `
+// This functionality is not available on this platform.
+      `,
+  },
+  }}
+/>
+
 <TextBlock kind="note" visibleOn="ios">{`
-On iOS, it is also possible to blacklist or whitelist all instances of a **\`UIView\` subclass** or all \`UIView\` or all UIView subclasses that conform to a **protocol**. More on the topic in <Link to="/docs/sdk/api-reference?platform=ios#sensitive-data-hiding">API reference</Link>.
+On iOS, it is also possible to blacklist or whitelist all instances of a **\`UIView\` subclass** or all \`UIView\` or all UIView subclasses that conform to a **protocol**. More on the topic in <Link to="/docs/sdk/api-reference#sensitive-data-hiding">API reference</Link>.
+`}</TextBlock>
+
+<TextBlock>{`
+More on this topic in <Link to="/docs/sdk/api-reference?platform=android#blacklisted-classes">API reference</Link>.
 `}</TextBlock>
 
 ## Handling Content Presented in WebViews
 
 Native WebView classes are blacklisted by default. Its instances, or the whole class, must be whitelisted in order to record its content. However, a whitelisted web view can still contain HTML elements with sensitive content, and Smartlook thus does not record the HTML elements where the user enters data.
 
-<TextBlock>
-In order to change the default behavior, HTML elements that are not recorded by default can be whitelisted by assigning `smartlook-show` CSS class to them. Any HTML element that should not be recorded can be blacklisted by assigning `smartlook-hide` CSS class to it: <Link to="/docs/sdk/api-reference#sensitive-data-hiding">API reference</Link>.
-</TextBlock>
+<TextBlock>{`
+In order to change the default behavior, HTML elements that are not recorded by default can be whitelisted by assigning \`smartlook-show\` CSS class to them. Any HTML element that should not be recorded can be blacklisted by assigning \`smartlook-hide\` CSS class to it.
```

Comment:
Nevím jestli chceme uvádět HTML a CSS jako "kód", ale nahoře to tak je. Možná by nebylo špatný to mít sjednocený.